app/src/lib/stores/app-store.ts

```diff
@@ -3300,17 +3310,17 @@ export class AppStore extends TypedBaseStore<IAppState> {
         "This pull request's clone URL is not populated but should be",
         head.gitHubRepository.cloneURL
       )
-      const remoteName = forkPullRequestRemoteName(
```

Comment:
This rename isn't correct - `remoteName` is what this value is supposed to be, and the same thing with the function below.